### PR TITLE
fix semicolon expressions

### LINKED ISSUE #2: Await command that ends with semi-colon does not execute.
It appears that `await` commands that end with a semi-colon, or are wrapped in some way, do not execute correctly. Here is some sample output to demonstrate:

```
$ await-outside
> await Promise.resolve(42)
42
> await Promise.resolve(42);
...
...
...
...
> console.log(42);
42
undefined
> console.log(await Promise.resolve(42))
...
> console.log(await Promise.resolve(42));
...
...
>
```

Also reported here: https://github.com/sloria/local-repl/issues/22

### PR DESCRIPTION
**Pull request by danprince**
Closes #2

### CODE REVIEW

File: __tests__/index.js [hunk 1]
```diff
@@ -162,6 +162,15 @@ await 3
 
       `abc = global.__await_outside_result; void delete global.__await_outside_result;`
     ]);
+    // ignore trailing semicolons
+    // https://github.com/nfcampos/await-outside/issues/2
+    expect(wrapAwaitOutside("await fetch();")).toEqual([
```
Comment (nfcampos) [inline __tests__/index.js]: can we add a test case with several trailing semicolons., eg. `await fetch();;;`?

**Review by nfcampos — COMMENTED** [review]
LGTM, other than the additional test case I mentioned in the comment

File: __tests__/index.js (hunk 1 above)
Comment (danprince) [reply __tests__/index.js]: All set!

**Comment by nfcampos** [comment]
@danprince Thank you! I'll publish a new version now

**Comment by nfcampos** [comment]
@danprince it's published as 3.0.0 (bumped major version because I also updated deps to fix vulnerabilities and that has changed the minimum node version from 6 to 8)

**Comment by danprince** [comment]
Thanks @nfcampos. Appreciate the speed!

**Pull request merged**